src/System.Diagnostics.Process/src/Resources/Strings.resx

```diff
@@ -187,6 +187,9 @@
   <data name="ProcessorAffinityNotSupported" xml:space="preserve">
     <value>Processor affinity for processes or threads is not supported.</value>
   </data>
+  <data name="ProcessStartInfoNotSupported" xml:space="preserve">
+    <value>ProcessStartInfo is not supported on this platform.</value>
```

Comment:
This isn't true. Most of PSI is supported, just not the properties related to identity.